dvc/render/vega.py

```diff
@@ -28,7 +28,7 @@ class PlotDataStructureError(DvcException):
     def __init__(self):
         super().__init__(
             "Plot data extraction failed. Please see "
-            "https://man.dvc.org/plot for supported data formats."
+            "https://man.dvc.org/plots for supported data formats."
```

Comment:
It looks like both links redirect to the same route, but I can confirm that the command is "plots" instead of "plot".

Reply:
Yeah it used to be `plot` I think so there's a redirect but the correct path is `plots` 🙂